test/manual/amp-consent-cmp.html

```diff
@@ -221,6 +221,14 @@
       PublisherLogo
     </div>
   </header>
+
+  <p>Set server forcePromptOnNext value to</p>
+
+  <ul>
+    <li><a href="../../get-consent-v1-set/1">True</a></li>
```

Comment:
nit / suggestion: it would be nice to make this a form that sends a simple xhr request, that way you can have a success sent state. 😄

Reply:
Great Idea! I'll change it.

Reply:
Done